starter-code/src/functions-and-arrays.js

```diff
@@ -59,6 +126,16 @@ var wordsFind = [
   'truth',
   'disobedience'
 ];
+var otherWord = prompt('Write a word to search');
+function findElements (wordsFind, otherWord){
+  for( i = 0; i< wordsFind.length; i++){
+    if(wordsFind[i] === otherWord){
+      return true;
+    }else{
+      return false;
+    } 
```

Comment:
No es correcto, solo iterará una vez, ya que tanto si cumple como si no cumple la condición, el `return` finalizará el bucle `for` y devolverá el booleano correspondiente. Mejor iguala una variable a `true` que tenga un valor predefinido `false`, así, si cumple la condición de que encuentre la palabra retornas `true` y si no, devuelves por defecto `false`